mantidproject/mantid
# LoadSampleShape algorithm for STL files

Description of work.

Creation of new algorithm to load and STL file into a MeshObject and place the resulting shape into the sample of the output workspace.

**To test:**

* Careful code review (issue #12828 only).
* Check that unit tests work and are adequate.
* Run the algorithm in Mantid to see that the dialog box is OK and it doesn't crash.
* Build and check the documentation.

Fixes #12828. 

**Release Notes** 
In New Algorithms of Framework in the release notes of 3.13.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
